pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -12,3 +12,11 @@ tasks:
       atools.simpleSky: SimpleDiaPlot
       python: |
         from lsst.analysis.tools.atools import *
+  analyzeTrailedDiaSrcCore:
+    class: lsst.analysis.tools.tasks.AssocDiaSrcDetectorVisitAnalysisTask
+    config:
+      connections.data: goodSeeingDiff_longTrailedSrc
+      atools.numDiaSourcesAll: NumDiaSourcesAllMetric
+      connections.outputName: trailedDiaSrcCore
+      python: |
+        from lsst.analysis.tools.atools import *
```

Comment:
Be sure to have a newline at the end of the file so the highly picky linter doesn't fail.